tests/playwright/src/fixtures/chat-fixtures.ts

```
@@ -34,14 +34,16 @@ interface TestFixtures {
 
 export const test = base.extend<TestFixtures, WorkerFixtures>({
   resource: [
-    async (_fixtures, use): Promise<void> => {
+    // eslint-disable-next-line no-empty-pattern
+    async ({}, use): Promise<void> => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)